.github/workflows/build.yml

```diff
@@ -12,8 +12,7 @@ concurrency:
 
 jobs:
   build:
-    if: always() && (needs.check.result == 'success' || needs.check.result == 'skipped')
```

Comment:
What was the `if` for anyway?

Reply:
when we had `requirements-sdp.txt`, there was a `check:` job that ensured it was populated IF the triggering event was a release (always passed otherwise)